squad/core/models.py

```diff
@@ -409,20 +409,28 @@ def overall(self):
 
 
 class TestSummaryBase(object):
+
     @property
     def tests_total(self):
         return self.tests_pass + self.tests_fail + self.tests_skip
 
-    @property
-    def pass_percentage(self):
-        if self.tests_pass > 0:
-            return 100 * (float(self.tests_pass) / float(self.tests_total))
+    def __percent__(self, ntests):
+        if ntests > 0:
```

Comment:
hmm, shouldn't that be 'if self.tests_total > 0' ? If ntests is 0, the result will still be 0 if the denominator is not 0.

Reply:
if any of the subsets (pass/fail/skip) is > 0, by definition tests_total will also be > 0